script.js

```diff
@@ -0,0 +1,11 @@
+let navToggle = document.getElementsByClassName("navToggle")[0]
+let navLinks = document.getElementsByClassName("navLinks")[0]
+
+navToggle.addEventListener("click", menuDown);
+function menuDown(){
+    if(navLinks.style.display === "none"){
+        navLinks.style.display="block"
+    }else{
+        navLinks.style.display="none"
+    }
+}
```

Comment:
Could you please try to reproduce this flow:
1. Open the html page in browser.
2. Narrow down so list from header becomes "burger" button.
3. Click it so menu is opened.
4. Go back to full screen mode

The list will not become aligned back